src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,10 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export const getFilteredTodos = (todos: Todo[], todoStatus: Status) => {
+  switch (todoStatus) {
+    case 'active': return todos.filter(todo => !todo.completed);
+    case 'completed': return todos.filter(todo => todo.completed);
+    case 'all': default: return todos;
```

Comment:
Use a enum for statuses
```suggestion
    case Status.active:
      return todos.filter(todo => !todo.completed);
    
    case Status.completed:
      return todos.filter(todo => todo.completed);
      
    case Status.all:
    default:
      return todos;
```